src/npe2/manifest/contributions/_contributions.py

```diff
@@ -36,8 +36,8 @@ class ContributionPoints(BaseModel):
     widgets: Optional[List[WidgetContribution]]
     sample_data: Optional[List[SampleDataContribution]]
     themes: Optional[List[ThemeContribution]]
-    menus: Dict[str, List[MenuItem]] = Field(default_factory=dict, hide_docs=True)
-    submenus: Optional[List[SubmenuContribution]] = Field(None, hide_docs=True)
+    menus: Dict[str, List[MenuItem]] = Field(default_factory=dict, hide_docs=False)
+    submenus: Optional[List[SubmenuContribution]] = Field(None, hide_docs=False)
```

Comment:
I don't fully understand, but wouldn't it be possible to take out the Field assignment? since hide_docs is True by default
```suggestion
    menus: Dict[str, List[MenuItem]]
    submenus: Optional[List[SubmenuContribution]]
```